hamster/menu.py

```diff
@@ -102,6 +150,7 @@ def edit_home_path(self, _):
         except Exception as e:
             rumps.alert("Error", e)
 
+    @track(uuid, "Recent Test Plans")
```

Comment:
Wouldn't this disclose the folder path of user directory? 

Ex: `/Users/optimusprime/setup/jmeterProjects/sample.jmx`